crates/router/src/services/api.rs

```diff
@@ -1147,6 +1153,92 @@ pub fn build_redirection_form(
+                            })
+                            </script>
+                            "#))
+
+
+                        h3 style="text-align: center;" { "Please wait while we process your payment..." }
+                    }
+
+                    (PreEscaped(format!("<script>
```

Comment:
You can use raw string literals to improve readability.

Reply:
Will take up this change in a different PR